{"nick":"Jagoan","date":"2016-07-02T01:38:46.558Z","type":"join"}
{"nick":"cooltpt","date":"2016-07-02T02:28:33.383Z","type":"join"}
{"nick":"cooltpt","message":"anyone home?","date":"2016-07-02T02:28:44.729Z","type":"message"}
{"nick":"cooltpt","message":"from the docs, it says if I add the handler -> app.use(function(err, req, res, next) {}); <- it's suppose to be called if the others don't handle the request","date":"2016-07-02T02:29:10.934Z","type":"message"}
{"nick":"cooltpt","message":"but, i don't see that happening.. just \"Cannot GET /api/test1x\" for example","date":"2016-07-02T02:29:45.502Z","type":"message"}
{"nick":"cooltpt","message":"from the docs, it says if I add the handler -> app.use(function(err, req, res, next) {}); <- it's suppose to be called if the others don't handle the request","date":"2016-07-02T02:30:03.834Z","type":"message"}
{"nick":"cooltpt","message":"but, i don't see that happening.. just \"Cannot GET /api/test1x\" for example","date":"2016-07-02T02:30:11.000Z","type":"message"}
{"nick":"gojira_","reason":"*.net *.split","date":"2016-07-02T05:04:52.576Z","type":"quit"}
{"nick":"cooltpt","reason":"Quit: Leaving","date":"2016-07-02T05:15:34.505Z","type":"quit"}
{"nick":"gojira_","date":"2016-07-02T13:10:00.236Z","type":"join"}
{"nick":"eyi","date":"2016-07-02T17:06:50.652Z","type":"join"}
